src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -146,6 +148,7 @@ class FixedwingAttitudeControl
 	struct vehicle_land_detected_s			_vehicle_land_detected;	/**< vehicle land detected */
 	struct vehicle_rates_setpoint_s			_rates_sp;	/* attitude rates setpoint */
 	struct vehicle_status_s				_vehicle_status;	/**< vehicle status */
+	struct fw_att_ctrl_status_s 			_fw_att_ctrl_status;	/**< controller status */
```

Comment:
There's no need to keep the struct in the class if it's fully populated and published together below.